app/javascript/app/layouts/root/root.ducks.js

```diff
@@ -0,0 +1,12 @@
+// Navigation pre-fetching thunks
+export async function getSectionsThunk(dispatch) {
+  const sections = await fetch(
+    'http://localhost:3000/api/v1/sections?includes[]=categories&includes[]=targets'
```

Comment:
Move the first part of the url to a constant in .env

Reply:
Also, let's check if we have the sections before fetching.
It would be nice to fetch sections, categories and targets separately to be able to check this. And maybe to override this condition when we want to. Let's talk about this.